optlang/glpk_interface.py

```diff
@@ -579,6 +579,9 @@ def __setstate__(self, repr_dict):
         if repr_dict['glpk_status'] == 'optimal':
             self.optimize()  # since the start is an optimal solution, nothing will happen here
 
```

Comment:
You might also need to delete self._variables/constraints/objective here since the Python GC does not resolve cyclical dependencies automatically for objects that have a __del__ method for some older Python versions I think...